django/db/models/base.py

```diff
@@ -40,6 +37,16 @@
 from django.utils.version import get_version
 
 
+class Deferred(object):
+    def __repr__(self):
+        return '<Deferred field>'
+
+    def __str__(self):
+        return '<Deferred field>'
```

Comment:
I searched a bit and couldn't find the exact ticket but the reported issue had something to do with Django's `lazy` logic and could be triggered even with ASCII only chars.